scripts/deploy/troubleshooting/get-volume-binding

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+if [ "$#" -ne 2 ]; then
+    echo "Usage: $0 <namespace> <deployment>"
+    exit 1
+fi
+
+NAMESPACE=$1
+DEPLOYMENT=$2
+
+get_pv_details() {
+    local pvc_name=$1
+    local pv_name
+    pv_name=$(kubectl get pvc -n "$NAMESPACE" "$pvc_name" -o jsonpath='{.spec.volumeName}' 2>/dev/null)
+    if [ -n "$pv_name" ]; then
+        kubectl get pv "$pv_name" -o json 2>/dev/null | jq '{
+            pvName: .metadata.name,
+            pvSize: .spec.capacity.storage,
+            storageClass: .spec.storageClassName,
+            pvAccessModes: .spec.accessModes
+        }'
+    else
+        echo "{}"
+    fi
```

Comment:
Do we want to continue executing if pv is not found? If not, consider adding error message here like you've done in other places.

Reply:
PV doesn't exist in some cases and that's expected for older deployments as well as single node deploys on k3s.  We have cases where pv/pvc are not used.